### DISCUSSION TITLE
Favorites, lists, tags?

### DISCUSSION THREAD

**Discussion by joemasilotti** [joemasilotti/railsdevs.com#498] [Ideas]
A few businesses have reached out expressing frustration in keeping track of the developers they want to contact. Some were lost with a huge list of folks they've contacted and no real understanding of who they need to follow up with. Others were overwhelmed with having to find/filter folks _and_ message them in one go (instead of splitting the work into two chunks).

After diving in a bit more with these folks it sounds like some sort of favorites, bookmarks, or tags could help streamline these workflows a bit.

## Options

Some different approaches that could be taken to help businesses.

### Option 1: Bookmark/favorite

This idea involves adding a bookmark/favorite button to developer profiles. Then consolidating these profiles in a list somewhere else in the app (maybe `/developers/saved`?). Each developer would have a link to their profile, the conversation, and a "remove" button.

I like this idea because it is fairly low effort to implement and doesn't require a lot of forced behavior. Businesses are free to use this list however they want. It also could create a good onboarding flow by having them explore the developers before sending any messages.

### Option 2: Lists

Similar to Option 1, but with more than one list. Added complexity comes from this being a many-to-many relationship and all the extra UX around managing it all.

I like this idea because it creates even more of a choose-your-own-adventure workflow but a business can customize it to their needs. But I'm not sure it's worth the additional work on top of Option 1.

### Option 3: Notes

Similar to HEY, allow private notes to be taken for a candidate. This could appear in the conversation thread in a different color to show that it is private to the business.

I like that this keeps a business using the platform a bit longer but I'm not sure if it actually solves the problem. Perhaps each candidate has _one_ note that lives at a higher level, an index view of sorts.

## Variants

Tweaks to the above options that could be added in incrementally.

## Variant 1: Notes

Like Option 3 above but each candidate has a single note. This could appear next to the developer profile in an index view of a list.

## Variant 2: Admin-managed input

I email a list of developers to every business that signs up for the $299/month plan. What if this appeared inside the app instead? Then a business would get a notification and could see all of the profiles right inside the app, instead of bouncing back and forth to email.

This could also coincide with Variant 1 to allow admins to add custom notes right in the page.

---

Right now this is serving as some open notes but, as always, I'd love to hear your input!

**Comment by silva96** [comment]
Have you consider the Kanban option?

You can move devs from one state to another by dragging them to other columns of the board.

Anyone that you contact could be in the most left column

https://getonbrd.com does it like that.

![IMG-20220616-WA0004](https://user-images.githubusercontent.com/4019924/174204597-bf2b71c3-b3f9-466f-a76b-afcef17692fc.jpg)

**Reply by joemasilotti** [reply]
Ohh, intersting! I like the idea of moving prospects through different stages of the process. But I'm not sure a full-on kanban board is worth it (to start). Maybe somewhere in the middle that groups candidates based on their status. Could be a fun Hotwire project, too!

Thanks for the link.
